packages/messaging/src/interfaces/public-types.ts

```diff
@@ -55,7 +55,8 @@ export interface NotificationPayload {
  */
 export interface FcmOptions {
   /**
-   * The link to open when the user clicks on the notification.
+   * The link to open when the user clicks on the notification.  For all URL values, HTTPS is required. 
+   * It is required that the hostname of this property be the same as the hostname of the registered service worker handling the notification.
```

Comment:
Suggest "The hostname the hostname of this property must be the same as . . ."

Reply:
Amended.Thanks for the feedback